### PR TITLE
Add Neobeam to list of community themes

### PR DESCRIPTION
**Pull request by mikael-ros**
This is my own theme. I saw someone else adding their own, so I figured I would too.

## Why awesome?
- It has a decent amount of recognition already, with 12 stars
- It provides a "sensible" default theme, so to speak. It's nothing fancy whilst being attractive enough
- It is a highly flexible theme, with plenty of CSS variables. It adjusts it's entire color scheme based on a single CSS variable
- The CSS is as clean as I could achieve

### CODE REVIEW

**Comment by yhatt** [comment]
Please make individual pull requests for each suggestions.
https://github.com/marp-team/awesome-marp/blob/690c76163ea71bb5a05e733a76b3bdc579260628/CONTRIBUTING.md?plain=1#L40

**Comment by mikael-ros** [comment]
Sorry, my first time contributing to a repository I don't own. I'll change this PR to only be for Neobeam, and submit the other suggestion seperately. Thank you for your patience :)

**Comment by mikael-ros** [comment]
I have now submitted it seperately. Take the time you need for reviewing them. I apologize for my mess. Again, it is perfectly okay if you choose to decline either of them.

**Comment by yhatt** [comment]
[Linter for Awesome lists](https://github.com/sindresorhus/awesome-lint) is not passing. Check whether URL is correct.
https://github.com/marp-team/awesome-marp/actions/runs/11564729527/job/32190787997#step:6:13

**Comment by mikael-ros** [comment]
Sorry, I missed the ``:`` before ``//`` in the URL. It should pass now.

File: README.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ A curated list of awesome things related to [Marp].
 - [Dracula](https://draculatheme.com/marp) - A dark theme for Marp, a part of [Dracula Theme](https://draculatheme.com/) project.
 - [Graph Paper](https://rnd195.github.io/marp-community-themes/theme/graph_paper.html) - A theme with graph paper background.
 - [marpstyle](https://github.com/cunhapaulo/marpstyle) - The set of Marp themes, designed with simplicity and beauty as first class prerequisites.
+- [Neobeam](https://www.github.com/mikael-ros/neobeam) - A modern take on LaTeX beamer.
```
Comment (yhatt) [inline README.md]:
```suggestion
- [Neobeam](https://github.com/mikael-ros/neobeam) - A modern take on LaTeX beamer.
```

**Review by yhatt — COMMENTED** [review]
Thanks for your first-time contribution 😄

**Comment by mikael-ros** [comment]
Thank you :)

**Review by yhatt — APPROVED** [review]

**Pull request merged**